README.md

```diff
@@ -14,11 +15,11 @@ and Pereira, J. M. B.<sup>3</sup>
 de Janeiro, RJ, Brazil.*
 
 This paper has been published in 
-[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](https://doi.org/10.1002/2016GC006329).
+[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](http://agupubs.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1525-2027).
 
-doi: [10.1002/2016GC006329](https://doi.org/10.1002/2016GC006329)
+doi: [10.1002/2016GC006329](http://onlinelibrary.wiley.com/doi/10.1002/2016GC006329/abstract)
 
-A PDF version is available at [pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html](http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html)
+A PDF version is available at [pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf](http://www.pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf)
```

Comment:
No. You link to the publication page.

You changed `http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html` to `http://www.pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf` for some reason. Why?